### PR TITLE
docs: minor improvements to readme and license

### PR DESCRIPTION
**Pull request by eltrevii**
## Changes
title

## Status

- [x] Follow the installation from <a href="https://github.com/Muunatic/RyU#how-to-use">**README**</a>.
- [x] Add or edit tests to reflect the change.
- [x] Run npm test.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-© 2021 Muunatic. All Rights Reserved.
+© 2023 Muunatic. All Rights Reserved.
```
Comment (HarunamiYaki) [inline LICENSE]:
```suggestion
© 2021-2023 Muunatic. All Rights Reserved.
```

**Review by HarunamiYaki — COMMENTED** [review]
Thank you for your initial contribution! Your pull request looks great, and we're excited to merge it!

Before we proceed, could you please adjust the target branch of this pull request to [`development`](https://github.com/Muunatic/RyU/tree/develop) instead of [`v5`](https://github.com/Muunatic/RyU/tree/v5)? This will ensure that your changes are aligned with our current development. Also don't forget to use conventional commit like `docs: Update README and LICENSE` for both your pull request title and commit messages. Additionally, I would appreciate it if you could take a moment to review the slight changes I've made. Thanks a lot! ❤ @eltrevii

File: LICENSE (hunk 1 above)
Comment (eltrevii) [reply LICENSE:1]: ill take a look later

**Comment by eltrevii** [comment]
how do i change the branch

**Comment by HarunamiYaki** [comment]
> how do i change the branch

You can navigate to the **"Edit"** option within your pull request interface and proceed to modify the base branch to the "develop" branch. Additionally, for the purpose of altering your semantic commit message, you may employ the command `$ git rebase -i HEAD~2` to edit your commit message to `docs: Update README and LICENSE`. If you dont familiar with git I suggest you to refork this repository and create a new pull request. Thanks! @eltrevii

**Comment by Muunatic** [comment]
thanks for improvement let me change the branch and adjust some commit message, and i'll approve this PRs after adjust some changes

**Review by Muunatic — APPROVED** [review]
LGTM!
thanks @eltrevii

**Comment by eltrevii** [comment]
ready to merge

**Pull request merged**